document_url/wizard/document_url.py

```diff
@@ -2,41 +2,39 @@
 # © 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
 #                       Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
 # © 2016 ACSONE SA/NV (<http://acsone.eu>)
-from openerp.osv import fields, orm
+from odoo import models, fields, api
 try:
     # Python 3
     from urllib import parse as urlparse
 except:
     from urlparse import urlparse
 
 
-class AddUrlWizard(orm.TransientModel):
+class AddUrlWizard(models.TransientModel):
     _name = 'ir.attachment.add_url'
 
-    _columns = {
-        'name': fields.char('Name', required=True),
-        'url': fields.char('URL', required=True),
-    }
+    name = fields.Char('Name', required=True)
+    url = fields.Char('URL', required=True)
 
-    def action_add_url(self, cr, uid, ids, context=None):
+    @api.multi
+    def action_add_url(self):
         """Adds the URL with the given name as an ir.attachment record."""
-        if context is None:
-            context = {}
-        if not context.get('active_model'):
+
+        context = self.env.context
+        if not context['active_model']:
```

Comment:
Keep the original `context.get` - the new form will error if the looked up key is not found in the context.